zipline/utils/calendars/trading_calendar.py

```diff
@@ -460,7 +460,10 @@ def minutes_for_session(self, session_label):
         pd.DateTimeIndex
             All the minutes for the given session.
         """
-        return self.minutes_in_range(*self.schedule.loc[session_label])
+        return self.minutes_in_range(
```

Comment:
If we're willing to assume that trading days are contiguous (i.e., that there are no holes in the calendar), then we can skip `minutes_in_range` entirely and just return `pd.date_range(start_minute, end_minute)` here.